src/commands/application/instances/connect.rs

```diff
@@ -18,16 +24,31 @@ struct Status {
     service: bool,
 }
 
-pub async fn handle_connect(context: Context, name: &str, port: &u16) -> Result<bool, CliError> {
+struct KubernetesPod {
+    name: String,
+    ready: bool,
+    is_livebook: Option<bool>,
+}
+
+pub async fn handle_connect(context: Context) -> Result<bool, CliError> {
     let spinner_style =
         ProgressStyle::with_template("{prefix:.bold.dim} {spinner} {wide_msg}").unwrap();
 
-    let (namespace, version, instance_name) = parse_name(name)?;
+    let namespace = match select_deployment_namespace()? {
```

Comment:
I think before we should the selection, we should show the current application first:
```
Current application: `wukong-ci-mock`.

? Please choose the namespace you want to connect to ›
❯ Prod
  Staging 
```

(You can refer to `wukong deployment execute`